src/Twitter.php

```diff
@@ -803,14 +804,21 @@ public function friendshipsCreate($id, array $params = []) : Response
      *
      * Returns the next cursor if there are more to be returned.
      *
-     * @param int|string $id
+     * $id may be one of any of the following:
+     *
+     * - a single user identifier
+     * - a single screen name
+     * - a list of user identifiers
```

Comment:
Yes, because the previous API was about a single identifier. The problem is child classes, not really the twitter API.